lib/reline/line_editor.rb

```diff
@@ -1089,6 +1089,7 @@ def input_key(key)
       end
     end
     if key.char.nil?
+      process_insert(force: true)
```

Comment:
Is `process_insert` needed in `key.char.nil? == true`?

Reply:
It's needed but not related to this pull request. I'll open another one with test code